src/ir/item.rs

```diff
@@ -825,6 +826,50 @@ impl Item {
             _ => None,
         }
     }
+
+    /// Returns whether the item is a constified module enum
+    fn is_constified_enum_module(&self, ctx: &BindgenContext) -> bool {
```

Comment:
The check needs to "peel back" one layer of `Alias`es. The issue with the `resolve()` function is that it resolves through all of the `Alias`es. 

I think that implementation should remain as is.